tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/EngSys/CodeownersTools/CodeownersTools.cs

```diff
@@ -145,7 +143,7 @@ public override async Task HandleCommand(InvocationContext ctx, CancellationToke
             }
         }
 
-        [McpServerTool(Name = "azsdk_engsys_codeowner_update"), Description("Adds or deletes codeowners for a given service label or path in a repo.")]
+        [McpServerTool(Name = "azsdk_engsys_codeowner_update"), Description("Adds or deletes codeowners for a given service label or path in a repo. When isAdding is false, the inputted users will be removed.")]
```

Comment:
I should have commented earlier, but why not have a `add_codeowner` and `remove_codeowner` tool? It feels a bit odd to have an update tool that defaults to a removal based on a parameter that's hidden within a list of 8 parameters. It seems easy for an mcp client or a user to miss and accidentally remove someone (when I assume the majority scenario is adding) We could avoid this extra description by just having two tools.

Also do we always expect only one of service label or path to be set? I would assume the mcp client will usually try to set both as it's not very good at understanding how to call parameters without much description.